src/mlpack/core/hpt/hpt.hpp

```diff
@@ -105,6 +106,26 @@ class HyperParameterTuner
   template<typename... CVArgs>
   HyperParameterTuner(const CVArgs& ...args);
 
+  //! Access and modify the optimizer.
+  OptimizerType& Optimizer() { return optimizer; }
+
+  /**
+   * Access and modify relative increase of arguments for calculation of partial
+   * derivatives (by the definition) in gradient-based optimization. The
+   * exact increase for some particular argument is equal to the absolute
+   * value of the argument multiplied by the relative increase (see also the
+   * documentation for MinDelta()).
+   */
+  double& RelativeDelta() { return relativeDelta; }
+
+  /**
+   * Access and modify minimum increase of arguments for calculation of partial
+   * derivatives (by the definition) in gradient-based optimization. This value
+   * is going to be used when it is greater than the increase calculated with
+   * the rules described in the documentation for RelativeDelta().
+   */
+  double& MinDelta() { return minDelta; }
```

Comment:
Ok. When I designed the interface I assumed that`HyperParameterTuner` objects would not be used with the `const` qualifier.